docker/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Dockerizing EIDA NG Mediator, Federator and StationLite images
+# Following the instructions from:
+#
+# https://github.com/EIDA/mediatorws
+#
+# Build Container:
+# docker build -t eida-federator:1.0 .
+#
+# Run Container:
+# docker run [-d] [--rm] [--name eida-federator] -p 8080:80 eidang_federator:1.0
+#
+# Modify running container:
+# docker exec -it eida-federator /bin/bash
+
+# Base image
+FROM ubuntu:16.04 
```

Comment:
@Jollyfant, did you also test it under ubuntu:18.04? I'd prefer the most recent LTS version.

Reply:
Will try it